app/views/admin/claims/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-xl">
+      Claims
+    </h1>
+
+    <% if @claims.count > 0 %>
+
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Reference</th>
+            <th scope="col" class="govuk-table__header">Applicant Name</th>
+            <th scope="col" class="govuk-table__header"></th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+          <%= render partial: "claim", collection: @claims %>
```

Comment:
This is coupled to the headers. What was the motivation for making it a partial?

Reply:
To make the markup simpler really. I've never thought about not using a partial for table rows.

Reply:
I've always been on the fence regarding using a partial to render a collection. I think if the row markup is simple enough I prefer the convenience of having it all in one template. For more complex pages and markup it can make sense to pull them out to a partial.

Incidentally there is also the Rails convention-over-configuration shorthand for this call:

```suggestion
          <%= render @claims %>
```